extensions/SampleExtension/Program.cs

```diff
@@ -1,35 +1,72 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-using System;
-using System.Threading;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
 using Microsoft.Windows.DevHome.SDK;
+using SampleExtension.Providers;
+using Serilog;
 
 namespace SampleExtension;
 
 public class Program
 {
     [MTAThread]
-    public static void Main(string[] args)
+    public static void Main([System.Runtime.InteropServices.WindowsRuntime.ReadOnlyArray] string[] args)
```

Comment:
Should we include changes from #3445?

Reply:
I evaluated those changes and determined they do not apply to the sample (at least not yet). There is no async call in the sample, and the argument change is for a different set of arguments, so the current is correct.